### PR TITLE
STENCIL-3559 403 error message

### PR DESCRIPTION
**Pull request by mcampa**
Change the 403 page message to be more friendly

https://jira.bigcommerce.com/browse/STENCIL-3559

## before
image

## after
image

@bigcommerce/stencil-team

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -778,8 +778,9 @@
         "message": "Store is currently unavailable, please try again in a few minutes"
     },
     "forbidden": {
-        "page_heading": "403 Error - Forbidden",
-        "message": "You do not have permission to access this page."
+        "page_heading": "Sorry! Please log in to continue",
+        "message": "You do not have permission to access this page. If you were logged in, ",
+        "sign_in": "please sign in"
```
Comment (junedkazi) [inline lang/en.json]: `If you were logged in, please sign in` does not make sense to me. Is this the actual copy or this is going to change ?

Comment (mcampa) [inline lang/en.json]: https://jira.bigcommerce.com/browse/STENCIL-3559?focusedCommentId=402224&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-402224

```
Sorry! Please log in to continue
If you were logged in, please log back in
to resume your work in a new session.
```

Comment (junedkazi) [inline lang/en.json]: So I  think it is missing `to resume your work in a new session.` ?

**Comment by mcampa** [comment]
image

**Comment by nickdengler** [comment]
Thanks, I have a PR ready with the fix for this. When this goes out, we'll need to merge it at the same time.

**Comment by nickdengler** [comment]
Here: https://github.com/bigcommerce/bigcommerce/pull/20375

**Pull request merged**